appveyor.yml

```diff
@@ -29,8 +29,6 @@ test_script:
   - OpenCover\tools\OpenCover.Console.exe -register:user -returntargetcode -target:"%xunit20%\xunit.console.x86.exe" -targetargs:"\"c:\projects\nlogweb\tests\NLog.Web.Tests\bin\Release\net452\NLog.Web.Tests.dll\" -appveyor -noshadow"  -returntargetcode -filter:"+[NLog.Web]*" -excludebyattribute:*.ExcludeFromCodeCoverage* -hideskipped:All -oldstyle -output:coverage.xml
```

Comment:
Maybe also remove the use of `OpenCover.Console.exe` and `nuget.exe install OpenCover` (just above)

Reply:
completed

Reply:
i have done that, the build seems to now test all .NET platforms.